scripts/jenkins-mender-convert-build.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+LINARO_COMPILER=gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
+COMPRESSED_COMPILER=${LINARO_COMPILER}.tar.xz
+LINARO_BINARY_PATH=${LINARO_COMPILER}/bin
+
+DEBIAN_IMAGE=bone-debian-9.4-iot-armhf-2018-06-17-4gb.img
+COMPRESSED_DEBIAN=${DEBIAN_IMAGE}.xz
+CONVERTED_DEBIAN_IMAGE=mender_debian_converted.sdimg
+
+RASPBIAN_FILE_NAME=2018-06-27-raspbian-stretch-lite
```

Comment:
There is newer Raspian image from 2018-11-13. It was checked that it is correctly converted with `mender-convert`.